save-deploy/README.md

```diff
@@ -78,13 +78,18 @@ If you run on windows, dependency `save-agent` is omitted because of problems wi
 To run on windows you need to compile save-agent on wsl and put saveAgentDistroFilepath to %USERPROFILE%\.gradle\gradle.properties <br/>
 For example: <br/> 
 `saveAgentDistroFilepath=file:\\\\\\\\wsl$\\Ubuntu\\home\\username\\projects\\save-cloud\\save-agent\\build\\libs\\save-agent-0.3.0-alpha.0.48+1c1fd41-distribution.jar` <br/>
-If you need to test changes in save-cli also you can compile snapshot version of save-cli on wsl <br/> 
+If you need to test changes in save-cli also you can compile snapshot version of save-cli on wsl <br/>
 and set saveCliPath and saveCliVersion in %USERPROFILE%\.gradle\gradle.properties <br/>
 For example:<br/>
 `saveCliPath=file:\\\\\\\\wsl$\\Ubuntu\\home\\username\\projects\\save-cli\\save-cli\\build\\bin\\linuxX64\\releaseExecutable` <br/> 
 `saveCliVersion=0.4.0-alpha.0.42+78a24a8` <br/>
 the version corresponds to the file `save-0.4.0-alpha.0.42+78a24a8-linuxX64.kexe` <br/>
 
+#### Some workarounds:
+If setting `save-agnet`'s path in `gradle.properties` didn't help you (something doesn't work on Mac), you still can place all the files from `save-agent-*-distribution.jar` into `save-orchestrator/build/resources/main`.
```

Comment:
Could you provide more details (as a comment or in PR description, to keep it for future reference) on what exactly doesn't work on Mac? I.e. how you specified `-PsaveAgentDistroFilepath` option and other details

Reply:
I just did everything that @icemachined advised (taking into consideration `save-deploy/README.md`). Folder `save-orchestrator/build/agentDistro` is created, it contains `save-agent-*-distribution.jar`, but it never extracts to `save-orchestrator/build/resources` as well as `agentDistro` is never found in `-classpath`

Reply:
I got it via GitHub (since #949 we build save-agent using Actions). The point is that something doesn't work - save agent is located in `$buildDir/agentDistro`, but when creating a docker in runtime, I got `FileNotFoundException` because `save-agent.kexe` could not be found